### PR TITLE
vimPlugins: add aerial-nvim, telescope-fzf-native-nvim

### PR DESCRIPTION
**Pull request by onsails**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130490` at cdf9e3a3 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages built successfully:</summary>
  <ul>
    <li>spacevim</li>
    <li>tests.vim.test_vim_with_vim_nix</li>
    <li>vimPlugins.aerial-nvim</li>
    <li>vimPlugins.neovim-ayu</li>
    <li>vimPlugins.nvim-bufferline-lua</li>
    <li>vimPlugins.nvim-lspconfig</li>
    <li>vimPlugins.packer-nvim</li>
    <li>vimPlugins.rust-tools-nvim</li>
    <li>vimPlugins.telescope-fzf-native-nvim</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`configurePhase` should probably contain `runHook preConfigure` and `runHook postConfigure`.

Near pkgs/misc/vim-plugins/build-vim-plugin.nix:28:14:
```
   |
28 |       inherit unpackPhase configurePhase buildPhase addonInfo preInstall postInstall;
   |              ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/misc/vim-plugins/build-vim-plugin.nix:28:14:
```
   |
28 |       inherit unpackPhase configurePhase buildPhase addonInfo preInstall postInstall;
   |              ^
```
</li>
  </ul>
</details>

File: pkgs/misc/vim-plugins/vim-plugin-names [hunk 1]
```diff
@@ -484,6 +484,7 @@ nvim-lua/plenary.nvim
 nvim-lua/popup.nvim
 nvim-telescope/telescope-dap.nvim
 nvim-telescope/telescope-frecency.nvim
+nvim-telescope/telescope-fzf-native.nvim@main
 nvim-telescope/telescope-fzf-writer.nvim
 nvim-telescope/telescope-fzy-native.nvim
```
Comment (SuperSandro2000) [inline pkgs/misc/vim-plugins/vim-plugin-names:489]:
```suggestion
```

**Comment by teto** [comment]
seems like something is missing
```
|| E5113: Error while calling lua chunk: .../start/telescope-nvim/lua/telescope/_extensions/init.lua:11: This extension doesn't exist or is not installed: fzf
|| ...pack/nix/start/telescope-fzf-native-nvim/lua/fzf_lib.lua:11: /nix/store/83jg8k5li15s13n5pwlv01kcvvlgirr8-vim-pack-dir/pack/nix/start/telescope-fzf-native-nvim/lua/../build/libfzf.so: cannot open shared object file: No such file or directory
```

**Comment by onsails** [comment]
that's weird, it works for me. I've also checked that lsof -p <nvim_pid> lists libfzf.so.

**Pull request merged**